docs/swagger.yaml

```diff
@@ -204,96 +204,183 @@ components:
       x-examples:
         example-1:
           imageId: laptop_image4433
+    ImageId:
+      title: ImageId
+      type: object
+      properties:
+        id:
+          type: string
+          pattern: '^[0-9A-Za-z-_\.]+$'
+          maxLength: 128
+      required:
+        - id
+    CategoryEnum:
+      type: string
+      title: CategoryEnum
+      enum:
+        - general
+        - receipt
+        - orderConfirmation
```

Comment:
we normally use capitalised snake case for enums

Reply:
same bug here, i'll fix